pkg/manager/manager.go

```diff
@@ -27,13 +29,17 @@ import (
 	// to ensure that exec-entrypoint and run can make use of them.
 	"github.com/deislabs/ratify/config"
 	"github.com/deislabs/ratify/httpserver"
+	"github.com/deislabs/ratify/pkg/featureflag"
 	_ "github.com/deislabs/ratify/pkg/policyprovider/configpolicy"
 	_ "github.com/deislabs/ratify/pkg/referrerstore/oras"
+	"github.com/deislabs/ratify/pkg/utils"
 	_ "github.com/deislabs/ratify/pkg/verifier/notaryv2"
+	"github.com/open-policy-agent/cert-controller/pkg/rotator"
```

Comment:
As discussed offline, lets add links to OPA rotator docs in Ratify docs.  Curious, does rotator also support rotation in the case of customer provided cert?

Reply:
yes, rotator always checks if the current cert is valid, and rotate it if not even though it's provided by customers.